cmd/singularity/actions_test.go

```diff
@@ -156,6 +183,30 @@ func testSingularityExec(t *testing.T) {
 			}
 		}))
 	}
+
+	// Clean up
+	err = os.Remove(tmpfile.Name())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// test --no-home option
+	err = os.Chdir("/tmp")
+	if err != nil {
+		t.Fatal(err)
+	}
+	t.Run("noHome", test.WithoutPrivilege(func(t *testing.T) {
+		_, stderr, exitCode, err := imageExec(t, "exec", opts{noHome: true}, pwd+"/container.img", []string{"ls", "ld", "$HOME"})
+		if exitCode != 1 {
+			t.Log(stderr, err)
+			t.Fatalf("unexpected success running '%v'", strings.Join([]string{"ls", "ld", "$HOME"}, " "))
```

Comment:
Did you mean to have `-ld` instead of `ld` here?

Reply:
typo